### PR TITLE
Add blogpost for js-ipfs release 0.23.0

### PR DESCRIPTION
**Pull request by VictorBjelkholm**
Time to kick up the blog again. Very simple post describing the latest js-ipfs release.

### CODE REVIEW

**Comment by VictorBjelkholm** [comment]
@diasdavid @dignifiedquire please review this so we can publish this asap

File: src/23-js-ipfs-0-23/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+* Easier initialization
+* Datastore instead of blob-store
+* New tutorials
+* `jsipfs add --wrap-with-directory` feature
+* Support for unixfs sharding
+
+## Installation
+
+```bash
+npm install --save js-ipfs@0.23.0
```
Comment (dignifiedquire) [inline src/23-js-ipfs-0-23/index.md]: it's actually `npm install --save ipfs@0.23.0`

Comment (VictorBjelkholm) [inline src/23-js-ipfs-0-23/index.md]: Ofc, thanks! 👍

**Comment by VictorBjelkholm** [comment]
Rendered version for easier review: https://github.com/ipfs/blog/blob/e0cc4886c2708d16ee7d19fd1f61c5e630eee43f/src/23-js-ipfs-0-23/index.md

File: src/23-js-ipfs-0-23/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+> jsipfs add <filename> --wrap-with-directory
+> jsipfs add <filename -w # alias
+```
+
+# Exciting future (soon™)
+
+Here is a quick list of things that we will be heads down after this release
+
+- Circuit Relay - We are building Circuit Relay in order to let browser nodes to connect to any node in the network (e.g when two nodes do not have a common transport). Track the development and spec here -- https://github.com/libp2p/specs/tree/master/relay, https://github.com/libp2p/js-libp2p-circuit/pull/1. 
+- DHT - The DHT is the second last piece that needs to be built(the first being relay) in order to give browser nodes the ability to discover the location of content by themselves. Track dev at the repo https://github.com/libp2p/js-libp2p-dht
```
Comment (Kubuxu) [inline src/23-js-ipfs-0-23/index.md]: There should be space between built and `(`.

File: src/23-js-ipfs-0-23/index.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+breadcrumbs:
+  - {name: "23-js-ipfs-0-23", link: "./" }
+id: 23-js-ipfs-0-23
+date: 2017-03-24
+title: js-ipfs 0.23.0 released
+author: David Dias & Victor Bjelkholm
+collection: posts
+---
+
+Today we're happy to announce that we have released js-ipfs version 0.23.0
```
Comment (diasdavid) [inline src/23-js-ipfs-0-23/index.md]: missing a `.`

File: src/23-js-ipfs-0-23/index.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+## Highlights
+
+* DAG API (IPLD Support)
+* Interoperability with go-ipfs
+* Bootstrap nodes
+* Easier initialization
+* Datastore instead of blob-store
```
Comment (diasdavid) [inline src/23-js-ipfs-0-23/index.md]: just `Datastore`

**Comment by VictorBjelkholm** [comment]
Updated based on the reviews, will merge and publish this now.

**Pull request merged**